### DISCUSSION TITLE
Joint JS with React

### DISCUSSION THREAD

**Discussion by Kritica1** [clientIO/joint#2448] [Q&A]
### Introduction

I've just started with JointJS, I was trying to create a common wrapper for Nodes as I have different kinds of nodes whose properties differ like name, label, description, nodeType, icon but the size remains the same. 
It would be great if someone could give me a quick example of how can I do so.
Forgive me if my question seems silly. 
Thanks 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You can check the [Fault Tree Analysis](https://www.jointjs.com/demos/fault-tree-analysis) demo. We define a base [Event](https://github.com/clientIO/joint/blob/master/demo/fta/src/index.js#L3) class from which every other shape inherits from.
